vizro-core/src/vizro/_vizro.py

```diff
@@ -24,8 +25,13 @@ def __init__(self, **kwargs):
 
         """
         self.dash = dash.Dash(**kwargs, use_pages=True, pages_folder="", title="Vizro")
-        self.dash.config.external_stylesheets.append(
-            "https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined"
+        self.dash.config.external_stylesheets.extend(
```

Comment:
When we have other dbc themes working we should figure out how to handle this correctly, e.g. maybe we need to be inserting these to the beginning rather than end of `external_stylesheets` to make overrides work in the right order. But this is fine for now.

Reply:
Yes, I agree. I thought about that as well! For your information, usually, you load the Bootstrap theme and afterwards any custom CSS (ours, and theirs) because if the CSS selectors have the same specificity, the later one will be taken.